Makefile

```diff
@@ -1,6 +1,10 @@
+help: ## Print this help
+	@grep --no-filename -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sed 's/:.*##/·/' | sort | column -ts '·' -c 120
+
 all: cli server worker ## Build all binaries for host OS and CPU
 
 -include rules.mk
+-include lint.mk
```

Comment:
I moved the `lint-install` managed stuff here as I'd like to keep the Makefile as simplish as possible... lesson learned from osie's ;)